packages/zowe-explorer/package.json

```diff
@@ -1905,12 +1905,12 @@
     ]
   },
   "scripts": {
-    "build": "gulp build && yarn license && webpack --mode development",
+    "build": "gulp build && yarn license && ts-node ./scripts/getSecretsPrebuilds.ts && webpack --mode development",
```

Comment:
Do we want to try [copy-webpack-plugin](https://webpack.js.org/plugins/copy-webpack-plugin/) instead of the custom script?